components/donation/Donation.component.tsx

```diff
@@ -76,9 +84,14 @@ export default function DonationView({ nodeInfo, show }: ParentProps) {
               />
             </Box>
           </Grid>
-          <Grid>
+          <Grid
+            container
+            direction="row"
+            justifyContent="center"
+            alignItems="center"
+          >
             <QRCode
-              value={`${nodeInfo!.id}@${selectAddr.address}:${selectAddr.port}`}
+              value={`${bolt12}`}
```

Comment:
this is wrong! it is not the bolt12 that you are showing, but the invoice that it the BOL11.

For this reason you are not able to scan the QR, because the invoice has a bug that it is too big for QR code.